app/components/Message.tsx

```
@@ -56,10 +70,65 @@ export const Message = ({
+                        key={toolCallId}
+                        command={terminalInput.command}
+                        output={streamingOutput}
+                        isExecuting={true}
+                        status={status}
+                      />
+                    );
+                  case "output-available":
+                    const terminalOutputContent =
+                      terminalOutput.result.stdout +
+                        terminalOutput.result.stderr ||
+                      terminalOutput.result.error;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)